File: src/main/java/ru/kosdev/signfinder/FindResult.java
package ru.kosdev.signfinder;

import org.bytedeco.javacpp.opencv_core;


/**
 * Результат поиска места для подписи
 */
public class FindResult {

    private opencv_core.Mat transformedMat;
    private opencv_core.Rect rect;

    public FindResult(opencv_core.Mat transformedMat, opencv_core.Rect rect) {
        this.transformedMat = transformedMat;
        this.rect = rect;
    }

    /**
     * Место для подписи
     * @return
     */
    public opencv_core.Rect getRect() {
        return rect;
    }

    /**
     * Трансформированное изображение
     * @return
     */
    public opencv_core.Mat getTransformedMat() {
        return transformedMat;
    }
}

File: src/main/kotlin/ru/kosdev/signfinder/FindResult.kt
package ru.kosdev.signfinder

import org.bytedeco.javacpp.opencv_core

/**
 * Created by Константин on 26.09.2015.
 */
public data class FindResult (transformedMat :opencv_core.Mat, rect : opencv_core.Rect) {
    val transformedMat = transformedMat
    val rect = rect
}